Library/Homebrew/brew.rb

```diff
@@ -44,6 +44,9 @@
       help_flag = true
     elsif !cmd && !help_flag_list.include?(arg)
       cmd = ARGV.delete_at(i)
+    elsif help_flag_list.include?(arg) & cmd
```

Comment:
This is pretty opaque; can you elaborate on what this is doing?

Reply:
My bad, I meant `&&`. Fix incoming.
The two changes are meant to be read together: by ignoring the `HOMEBREW_HELP` variable I force Homebrew to detect if a help request was actually issued *to the current process*.
In the case above, when `brew tap homebrew/services` is issued, this case detects that *no* help flags or command are present and lets the tap proceed.